src/components/Work/Tabs/Cite.js

```diff
@@ -36,6 +42,39 @@ const TabsCite = props => {
     { label: "Use Statement", value: nulUseStatement }
   ];
 
+  const formats = [
+    {
+      id: "permalink",
+      label: "The ARK",
+      text: permalink,
+      style: {}
+    },
+    {
+      id: "apaFormat",
+      label: "APA Format",
+      text: `${admin_set}, ${nul}. (${date}). ${title}, Retrieved from ${item_link}`,
+      style: {}
+    },
+    {
+      id: "turabianFormat",
+      label: "Chicago/Turabian Format",
+      text: `${admin_set}, ${nul}. "${title}", ${collection_title} Accessed ${today}. ${item_link}`,
+      style: {}
+    },
+    {
+      id: "mlaFormat",
+      label: "MLA Format",
+      text: `${admin_set}, ${nul}. "${title}", ${collection_title} ${date}. ${window.location.origin}/items/${id}`,
+      style: {}
+    },
+    {
+      id: "wikiCitation",
+      label: "Wikipedia Citation",
+      text: `<ref name=NUL>{{cite web | url=${item_link} | title= ${title} (${date}) }} |author=Digital Collections, ${nul} |accessdate=${today} |publisher=${nul}, ${admin_set}}}</ref>`,
+      style: styles.monoSpace
+    }
+  ];
+
   return (
     <div data-testid="tab-content-cite">
       <div className="cite-group-col">
```

Comment:
Can we either remove this `cite-group-col` class, or adjust the style definition so that it's not just half a column?  Now that we're not using 2 columns for the cite tab content, we should let the content flow across the entire tab content area.